### ISSUE TITLE
[BUG]: v0.3.0 FileNotFoundError: /app/bin/classify_taxonomy

### ISSUE THREAD

**Issue by ptrebert** [ncbi/fcs#31]
*Describe the bug*
v0.3.0 `run_fcsgx.py` fails with
```
FileNotFoundError: [Errno 2] No such file or directory: '/app/bin/classify_taxonomy'
```

*To Reproduce*
See attached stderr output of cluster job for full command

*Software versions (please complete the following information):*
 - OS: CentOS 6
 - Singularity v3.5.2
 - FCS image v0.3.0
 - Python v3.8.3

*Log Files*
Can make a debug run if absolutely necessary
[fcstest.stderr.log](https://github.com/ncbi/fcs/files/10858106/fcstest.stderr.log)

*Additional context*
- Ealier (dev?) versions of the FCS pipeline were executed successfully on the same infrastructure (`fcs-gx.0.2.1.sif`)
- For the record: I shell'ed into the Singularity container and confirmed that `classify_taxonomy` is indeed present at the expected location and executable. So, unless you have seen this before, I would naively assume that this is not the true cause of the failure.

I am going to run a test with a smaller genome and the test database to see what happens.

+Peter

**Comment by boukn** [comment]
I can't reproduce this, but it likely has something to do with the bazel py-binary wrapper. 

Can you shell into the container and run `classsify_taxonomy --help ` 

Can you try also with an extra parameter, `--bind '/somewhere/safe/and/writable:/var/tmp'`

Are there values for the env vars TMP or TMPDIR?

**Comment by ptrebert** [comment]
Thanks for looking into this.

Yes, `classify_taxonomy [--help]` works (as already indicated above). I can mount a writable location to `/var/tmp` in the container and also create files w/o any problem. The default value for TMP is set by the cluster scheduler (PBS) as evident from the log file I had attached:

```
/var/tmp/pbs.5801516.hpc-batch/ [...]
```

That's the TMP location for the job local to the execution host. Is it possible that there was not enough space in TMP, (mis-) leading to the error? Can you give me a rough number how much space is needed in TMP to run the FCS pipeline?

**Comment by boukn** [comment]
Around 200 megs of space is needed. But if that was the problem it wouldnt really execute at all. Im now very confused.

**Comment by ptrebert** [comment]
Hm, I tested another example with half of the genome as input, and the run finished successfully. I am going to use the complete genome again, maybe the above was a weird machine error ...

**Comment by pstrope** [comment]
Glad it worked for you. Hope to get your feedback once you try out the newly released v0.4.0
